cabal.project

```diff
@@ -1 +1,23 @@
 packages: .
+
+-----------------------------------------------------------
+-- clash-compiler for GHC 9.10
+-----------------------------------------------------------
+
+source-repository-package
+  type: git
+  location: https://github.com/clash-lang/clash-compiler
+  tag: 15dc344dfa091de14c63759c0b6ea107ca0fa892
+  subdir: clash-lib
+
+source-repository-package
+  type: git
+  location: https://github.com/clash-lang/clash-compiler
+  tag: 15dc344dfa091de14c63759c0b6ea107ca0fa892
+  subdir: clash-prelude
+
+source-repository-package
+  type: git
+  location: https://github.com/clash-lang/clash-compiler
+  tag: 15dc344dfa091de14c63759c0b6ea107ca0fa892
+  subdir: clash-prelude-hedgehog
```

Comment:
```suggestion
-----------------------------------------------------------
-- clash-compiler for GHC 9.10
-----------------------------------------------------------
source-repository-package
  type: git
  location: https://github.com/clash-lang/clash-compiler
  tag: 15dc344dfa091de14c63759c0b6ea107ca0fa892
  subdir: clash-prelude
```

Reply:
Only `clash-prelude` is a dependency AFAIK. Was I wrong in my previous comment?

Reply:
Sigh, apologies; I had overlooked that bit of your comment. Fixed.